.github/workflows/lint-test.yaml

```diff
@@ -214,6 +229,32 @@ jobs:
             !/tmp/.tmp*/**/db/
           retention-days: 14
 
+  rust-network-perf-smoke-test:
```

Comment:
We should also have a forge test. Problem is we need a separate docker image because of the feature flag. Why do we use a feature over a config flag?

Reply:
Yeah, I'll add a forge test once this stack lands. Then hopefully we can utilize the network simulation stuff you're working on to see how the networking stack behaves.

> Why do we use a feature over a config flag?

This is mainly for security. With the feature enabled it allows bad peers to send performance messages, which may be huge or frequent. Without the feature, nodes will not accept these types of messages and simply drop them. I wanted to avoid the situation where someone "accidentally" enables this in the config and starts to spam the network.

Reply:
Understood from previous answer. I thought this was going to be a default feature at some point. Makes sense now. thanks!